### PR TITLE
docs: prefer Terraform for infra management

### PR DESCRIPTION
**Pull request by ace-n**
Part of #536; related to #559

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -69,6 +69,9 @@ Minimizing developer follow-up action is [helpful](#positive-helpful-feedback)!
 
 If no Google Cloud resources are needed, use [GitHub Actions](https://docs.github.com/en/actions) to drive automation. Otherwise use Cloud Build ([decision](docs/decisions/2021-05-static-analysis.md)).
 
+Infrastructure (including test fixtures) should be managed by Terraform wherever practical. In cases where resources **cannot** be managed by Terraform, this project recommends manual management instead
+of custom scripts/automation. ([decision](docs/decisions/2022-07-pet-test-fixtures.md))
```
Comment (grayside) [inline CONTRIBUTING.md:73]: This isn't quite true: we actively use scripts/automation in many places for one-time setup. Where we've avoided automation is project creation, and that's primarily a matter of security policy.

**Review by grayside — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (ace-n) [reply CONTRIBUTING.md]: Reworded; PTAL?

**Review by grayside — CHANGES_REQUESTED** [review]
Text looks good, but still pending removal of the "pet" terminology from the decision record & file.

**Comment by ace-n** [comment]
@grayside done. (Removed `pet` from the decision record's filename.)

**Review by grayside — APPROVED** [review]

**Pull request merged**